From f6861b82040b34ab5771d7532cd4c12237ba1e89 Mon Sep 17 00:00:00 2001
From: Richard Kiene <richard.kiene@joyent.com>
Date: Tue, 16 Oct 2018 14:51:32 +0000
Subject: [PATCH] TRITON-755 Want ability to filter CMON discovery results by
 tag

Reviewed by: Alex Wilson <alex.wilson@joyent.com>
Approved by: Alex Wilson <alex.wilson@joyent.com>
---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index e63b5fc..a71f7ff 100644
--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "restify-errors": "4.3.0",
     "sdc-scripts": "git+https://github.com/joyent/sdc-scripts.git#deefaef587ed3bee2706cb6e53ee3468e682932e",
     "sprintf-js": "1.0.3",
-    "triton-tags": "^1.1.2",
+    "triton-tags": "1.2.2",
     "uuid": "2.0.1",
     "vasync": "1.6.3"
   }
-- 
2.21.0

